apps/point-of-sale/src/views/CashierView.vue

```diff
@@ -58,6 +62,22 @@ enum PointOfSaleState {
 
 const currentState = ref(PointOfSaleState.DISPLAY_POS);
 
+const showTopUpWarning = ref(false);
+const hasShownTopUpWarning = ref(false);
+
+const shouldShowTopUpWarning = computed(() => {
+  const inDebt = cartStore.checkBuyerInDebt();
+  const notShown = !hasShownTopUpWarning.value;
+  return inDebt && notShown;

```

Comment:
You can simplify this into the following:
```suggestion
  return inDebt && !hasShownTopUpWarning.value;

```

Reply:
Simplified in last commit